imap_processing/codice/codice_l1a.py

```diff
@@ -102,19 +105,89 @@ def _generate_simulated_data(self, length_in_bits):
         random_bytes : bytes
             A list of random bytes to be used as simulated science data
         """
+        print("\tUsing simulated data")
```

Comment:
Change these to logs?
```suggestion
        logger.info("\tUsing simulated data")
```

Reply:
I plan to eventually turn all of these print statements into logging statement, but the print statements help me see how my code is doing while developing/testing, so I would like to leave them for now.

Reply:
OK, thats fine 👍 . Just to give another option, depending on how you're running your code you could set the log level at the header of this file, or via the command line options `-v` if you're running the command line program too.